### ISSUE TITLE
Error setting up entry... alexa_media (Echo) - HA 2022.3.3

### ISSUE THREAD

**Issue by andrewbpowers** [custom-components/alexa_media_player#1532]
After the update to version 2022.3.3 once again a new problem. But I am used to that by now.

```
This error originated from a custom integration.

Logger: homeassistant.config_entries
Source: custom_components/alexa_media/__init__.py:324 
Integration: Alexa Media Player (documentation, issues) 
First occurred: 11:45:54 AM (1 occurrences) 
Last logged: 11:45:54 AM

Error setting up entry xxx@xxx.com - amazon.com for alexa_media
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/alexapy/aiohttp/connector.py", line 1154, in _create_direct_connection
    hosts = await asyncio.shield(host_resolved)
  File "/usr/local/lib/python3.9/site-packages/alexapy/aiohttp/connector.py", line 880, in _resolve_host
    addrs = await self._resolver.resolve(host, port, family=self._family)
  File "/usr/local/lib/python3.9/site-packages/alexapy/aiohttp/resolver.py", line 33, in resolve
    infos = await self._loop.getaddrinfo(
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 856, in getaddrinfo
    return await self.run_in_executor(
  File "/usr/local/lib/python3.9/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.9/socket.py", line 954, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Try again

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/alexapy/helpers.py", line 137, in wrapper
    return await func(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/alexapy/alexalogin.py", line 574, in login
    if await self.test_loggedin(cookies):
  File "/usr/local/lib/python3.9/site-packages/alexapy/alexalogin.py", line 455, in test_loggedin
    get_resp = await self._session.get(
  File "/usr/local/lib/python3.9/site-packages/alexapy/aiohttp/client.py", line 535, in _request
    conn = await self._connector.connect(
  File "/usr/local/lib/python3.9/site-packages/alexapy/aiohttp/connector.py", line 542, in connect
    proto = await self._create_connection(req, traces, timeout)
  File "/usr/local/lib/python3.9/site-packages/alexapy/aiohttp/connector.py", line 907, in _create_connection
    _, proto = await self._create_direct_connection(req, traces, timeout)
  File "/usr/local/lib/python3.9/site-packages/alexapy/aiohttp/connector.py", line 1166, in _create_direct_connection
    raise ClientConnectorError(req.connection_key, exc) from exc
alexapy.aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host www.amazon.com:443 ssl:<ssl.SSLContext object at 0x7f7b344d40> [Try again]

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 335, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/alexa_media/__init__.py", line 324, in async_setup_entry
    await login.login(cookies=await login.load_cookie())
  File "/usr/local/lib/python3.9/site-packages/alexapy/helpers.py", line 147, in wrapper
    raise AlexapyConnectionError from ex
alexapy.errors.AlexapyConnectionError

```

Restarts of course have not changed anything.
It is so annoying!

**Comment by akerrichardt** [comment]
I have the same problem with all Alexa Media Player Entitles.  It all started after I updated to 2022.3.3.  After the failure I did a complete restore of my System to 2022.2.9 but still having the same problem.  I reinstall Alexa Media Player in HACS and do the Integration install.  Everything seemed to work until the final step when I receive the following error "Error 500 Internal Server Error Server got itself in trouble"  I tried to talk to Amazon support but that was a real waste of time,  Any help would be greatly appreciated.  Also my Alexa Guard is no longer working either.
